gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +0,0 @@
-distributionBase=GRADLE_USER_HOME
```

Comment:
Why do we need to remove `gradle-wrapper.jar` and `gradle-wrapper.properties`?

Reply:
Sorry, this was a misunderstanding on my side! I've restored both of these to what develop has, which should fix the codeql / circecli failures... as they have no gradle wrapper.